scripts/login_validation.js

```diff
@@ -0,0 +1,74 @@
+import { validateEmail, validatePassword } from './validation/validation.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const emailInput = document.getElementById('email');
+  const passwordInput = document.getElementById('password');
+  const loginButton = document.querySelector('.auth__button');
+  const loginForm = document.querySelector('form');
```

Comment:
💬 여담
제 생각에는 파일명을 통해 login 관련이라는 것을 알 수 있으므로 변수명에서 login을 빼도 될 것 같아요.

```suggestion
  const submitButton = document.querySelector('.auth__button');
  const form = document.querySelector('form');
```